### ISSUE TITLE
fault does not initialize clock

### ISSUE THREAD

**Issue by THofstee** [leonardt/fault#125]
If you wire the clock parameter to a testbench, I would assume that it works without having to do anything special to the clock. However fault does not initialize the clock, so the toggles leave it in an undefined (X) state if the initial state was X (as is the case in VCS).

**Comment by leonardt** [comment]
Yes, this is an issue with whether we want to have a known initial value, or have it be X like system verilog.

I would imagine it makes sense to have the clock be 0 (as you suggest), and perhaps have an option to have the clock start at X if there's some use case for that.
